apps/arena/lib/arena/game/skill.ex

```diff
@@ -351,6 +351,15 @@ defmodule Arena.Game.Skill do
     end
   end
 
+  defp do_effect_mechanics(_game_state, player, effect, {:speed_boost, speed_boost_params}) do
+    Process.send_after(self(), {:remove_speed_boost, player.id, speed_boost_params.amount}, effect.duration_ms)
+    Map.put(player, :speed, player.speed + speed_boost_params.amount)
+  end
+
+  defp do_effect_mechanics(_game_state, player, _effect, {:damage_up, _}) do
```

Comment:
What about the invisible one?

Reply:
It doesn't have mechanics... That's a pending refactor, but not something for now